pkg/serviceimport/map_test.go

```diff
@@ -40,12 +40,12 @@ var _ = Describe("ServiceImport Map", func() {
 	}
 
 	expectIPsNotFound := func(ns, service, cluster, localCluster string) {
-		_, found := serviceImportMap.GetIP(ns, service, cluster, localCluster, checkCluster, checkEndpoint)
+		_, found, _ := serviceImportMap.GetIP(ns, service, cluster, localCluster, checkCluster, checkEndpoint)
```

Comment:
I think we should verify `isLocal` and add a test for it.

Reply:
Will have to add a new test for it. Just adding an Expect for it here breaks the tests. Can add it later along with GetIP rework.